app/views/agents/_form.html.erb

```diff
@@ -134,15 +116,15 @@
     <div class="col-md-6">
       <div class="row">
         <div class="col-md-12">
-          <div class='well description'>
-            <%= @agent.html_description unless @agent.new_record? %>
+          <div class="well description">
+            <%= @form.html_description %>
```

Comment:
I don't believe it's needed. I did some testing and it never threw an error. I wrapped the call to `Agent#html_description` at the form level so even if `@agent` is nil, it will just return a `nil` which will be a blank text element.